src/commands/react-native-ci-cli.ts

```diff
@@ -7,10 +7,16 @@ import easUpdate from '../recipes/eas-update'
 import detox from '../recipes/detox'
 import isGitDirty from 'is-git-dirty'
 import sequentialPromiseMap from '../utils/sequentialPromiseMap'
-import { CycliToolbox, ProjectContext } from '../types'
+import { CycliRecipe, CycliToolbox, ProjectContext } from '../types'
 import messageFromError from '../utils/messageFromError'
+import { intersection } from 'lodash'
```

Comment:
I'm not sure if it's still a big deal, but some time ago there was big drama that lodash loads to many things and it's better to import it with `lodash/feature`

Reply:
valid, changed in 8562ff51b671434d519069645c812d81503a62aa